packages/components/atoms/f-image-tile/src/components/ImageTile.vue

```diff
@@ -69,14 +75,43 @@ export default {
         },
         imgSrc: {
             type: String,
-            default: null
+            default: ''
+        },
+        altText: {
+            type: String,
+            default: ''
+        },
+        fallbackImage: {
+            type: String,
+            default: ''
         }
     },
     data () {
         return {
             isToggleSelected: false
         };
     },
+    computed: {
+        /**
```

Comment:
I would remove the comment because we can rewrite the code so that it's self-explanatory. Unnecessary comments can be dangerous in the long run.